src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -908,9 +911,15 @@ public Vector2 ScreenPosition
         {
             get
             {
-                Vector2 temp = new Vector2(0.0f, 0.0f);
-                GetProperty(View.Property.ScreenPosition).Get(temp);
-                return temp;
+                using (PropertyValue screenPosition = GetProperty(View.Property.ScreenPosition)) 
+                {
+                    using (Vector2 vectorValue = new Vector2(0.0f, 0.0f))
+                    {
+                        Vector2 position = vectorValue;
+                        screenPosition?.Get(position);
```

Comment:
Although null check is cheap, but it does not necessary here compared with previous code.
Ditto to other APIs.